src/chrome/content/rules/Socialblade.com-falsemixed.xml

```diff
@@ -1,13 +0,0 @@
-
-<!--
-Disabled by https-everywhere-checker because:
-Fetch error: http://forums.socialblade.com/ => https://forums.socialblade.com/: (6, 'Could not resolve host: forums.socialblade.com')
-
-	See also Socialblade.com.xml
```

Comment:
I can't remove the reference to the deleted file from Socialblade.com.xml unless I will create another pull request.